final_prep/annotations.rb

```diff
@@ -3,11 +3,15 @@
 # Use the # to create a new comment
 
 # Build a Bear
-
+#This is defining the method with the name build_a_bear and assiging 5 arguments to it. \
 def build_a_bear(name, age, fur, clothes, special_power)
+  #This is assigning a string with the 'name' argument embedded in it, to the variable "greeting".
   greeting = "Hey partner! My name is #{name} - will you be my friend?!"
+  #This is assinging an Array holding the elements 'name', and 'age' t othe variable demographics.
   demographics = [name, age]
+  #This is assiging a string with the argument 'special_power' embedded in it, to the variable 'power_saying'.
   power_saying = "Did you know that I can #{special_power}?"
+  #This is creating a variable called 'built_bear' with a hash model assign to it with the attributes of a stuffed bear.
   built_bear = {
     'basic_info' => demographics,
```

Comment:
You can still go line by line here- it may seem like overkill, but it is important to keep track of where values come from. Were these declared as variables? Were they passed as params?